packages/ui/react-ui-thread/src/Thread/Thread.tsx

```diff
@@ -27,7 +27,8 @@ export const Thread = forwardRef<HTMLDivElement, ThreadProps>(
         className={mx(
           threadLayout,
           hoverableFocusedWithinControls,
-          'bg-[var(--surface-bg)] border-[color:var(--surface-separator)] border-bs border-be plb-1.5 attention attention-within attention-current [--controls-opacity:0]',
+          'plb-1.5 bg-[var(--surface-bg)] border-[color:var(--surface-separator)] border-bs border-be',
```

Comment:
```suggestion
          'plb-1.5 bg-[--surface-bg] border-[color:var(--surface-separator)] border-bs border-be',
```
(not sure about simplifying the color distinction, will investigate)